core/tests/protractor.conf.js

```diff
@@ -25,6 +25,10 @@ var suites = {
       'protractor_desktop/additionalPlayerFeatures.js'
     ],
 
+    adminTabFeatures: [
```

Comment:
This isn't a tab (it's full-page), right?

Reply:
Yeah right.Will change that.

Reply:
Done